crates/config/src/host_component.rs

```diff
@@ -0,0 +1,46 @@
+use std::sync::Arc;
+
+use crate::{Error, Key, Path, Resolver, Result};
+
+mod wit {
```

Comment:
I like this a lot, we might want to namespace the other bindings as well.

Reply:
I actually _had_ to (or at least this was the first solution that came to mind) because the generated bindings were named the same as the crate...

That said, yeah I think its a good pattern to make it obvious where the generated types are coming from.

Reply:
Alternatively, I think you could do `use <> as <>`.